reconcile/utils/terraform_client.py

```diff
@@ -246,6 +251,21 @@ def log_plan_diff(self, name, tf, enable_deletion):
                             )
         return disabled_deletion_detected, deleted_users, created_users
 
+    def deletion_approved(self, account_name, resource_type, resource_name):
+        account = self.accounts[account_name]
+        deletion_approvals = account.get('deletionApprovals')
+        if not deletion_approvals:
+            return False
+        now = datetime.utcnow()
+        for da in deletion_approvals:
+            expiration = datetime.strptime(da['expiration'], DATE_FORMAT)
+            if resource_type == da['type'] \
+                    and resource_name == da['name'] \
+                    and expiration <= now:
```

Comment:
the other way around now <= expiration

Reply:
yup, already caught that with testing. i started with a `if not confition: continue` structure, so that's where this bug came from.